packages/driver/src/reflection/queries/types.ts

```diff
@@ -127,7 +127,7 @@ export async function getTypes(
       material_scalars := (
         SELECT ScalarType
         FILTER
-          (.name LIKE 'std::%' OR .name LIKE 'cal::%')
+          (.name LIKE 'std::%' OR .name LIKE 'cal::%' OR .name LIKE 'ext::%')
```

Comment:
This works for now, but potentially a future extension may add non base-type scalars. So maybe the filter should check that the type has no non-abstract ancestors?

Reply:
> So maybe the filter should check that the type has no non-abstract ancestors?

Isn't that what `AND NOT .is_abstract` is doing on line 131? Or do you think there is some way to narrow the search space in `ext::` specifically?

Reply:
That's checking that the type itself isn't abstract. I meant checking the ancestors are all abstract as well like this:
```
select schema::ScalarType {
  name,
} filter not .abstract
  and not exists .enum_values
  and not exists (select .ancestors filter not .abstract)
```

Reply:
Oh yeah, maybe I don't understand after all: where would this query go?

Reply:
Instead of the current `material_scalars` query.

Reply:
Oh, so don't even filter on the name at all? We don't need the shape here, right? At the very least we also want at least the `id` since that's what we actually consume. I'll make the change and maybe add a test for `std`, `cal`, `ext`.

Reply:
@jaclarke ❓ [`992f2f0` (#686)](https://github.com/edgedb/edgedb-js/pull/686/commits/992f2f0e4cb35cccfb148794d77589d25d96a830)

Does this seem right? I left the name filtering in, but do you think that's unnecessary?